packages/gatsby-theme-store/src/sdk/lazyScript/useLazyScript.ts

```diff
@@ -3,11 +3,12 @@ import { useIdleEffect } from '@vtex/store-ui'
 interface Options {
   src: string
   id: string
+  useCors?: boolean
   // add script to the page after timeout ms after idleCallback fired
   timeout?: number
 }
 
-const registerScript = ({ id, src }: Options) => {
+const registerScript = ({ id, src, useCors }: Options) => {
```

Comment:
Isn't it better to allow people to overwrite any attribute? This way we are more hackable. Something like:

```
interface Options extends HTMLScriptAttributes {
  id: string
  timeout?: number
}

const registerScript = ({id, timeout, ...rest}: Options) => {
 // ...
 Object.keys.forEach(attribute => {
 document.setAttribute(attribute, rest[attribute])
 })
 // ...
}
```

Reply:
Niice! Great idea!